apps/src/StudioApp.js

```diff
@@ -391,7 +409,14 @@ StudioApp.prototype.init = function(config) {
     }
     var bubble = document.getElementById('bubble');
     dom.addClickTouchEvent(bubble, _.bind(function() {
-      this.showInstructions_(config.level, false);
+      var hintsToShow = this.authoredHints_.filter(function(hint){
+        return hint.alreadySeen === false;
+      });
```

Comment:
Nit: should be a space between (hint) and {

Also as an FYI, if we were writing this in ES6 it would be as simple as:
```
var hintsToShow = this.authoredHints_.filter(hint => hint.alreadySeen === false);